src/index.html

```diff
@@ -4,19 +4,100 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <title>Page Title</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
-    <title>Moyo header</title>
+
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="images\moyo_1.png"
+          alt="Logo"
```

Comment:
The `alt` attribute should describe the image content. For instance, if the image is a logo of a company, you should mention the company's name in the `alt` attribute. This will improve your page's SEO and help screen readers.